api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java

```diff
@@ -30,9 +33,6 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import java.util.Arrays;
-import java.util.Date;
-
 /**
```

Comment:
@moshonk @ningosi Please remove created by from all files in reporting module - that can be a separate PR